### ISSUE TITLE
Watch Files method is not working correctly

### ISSUE THREAD

**Issue by caokietbk** [deansyd/icloud_storage#2]
hi deanyuan,
Thank you for share this library.
I am using it with my project and I have problem when I listen to iCloud file change.
After I set up watch files listener in my device, I back up file in another device (using the same iCloud account). 
I see the listener can't listen that change.
Hope you check and fix this issue (if I am not wrong) soon and sorry for my bad English.

**Comment by deansyd** [comment]
Hi @caokietbk, have you checked whether the file has been synced by iCloud on the device listening for changes. You can do so by going to Settings -> Apple ID -> cloud -> Manage Storage, which shows a list of apps and the storage each of them takes up. Do you see any storage increase for your app on this device after uploading the file from another device? If not and if you are using the simulator, you need to manually trigger the sync by press Features -> Trigger iCloud Sync on your listening device. If you do see the storage increase and still experience the issue, could you please send through the code snippet, a step by step description of what you've done, accompanying with screen shot?

**Comment by caokietbk** [comment]
Hi @deansyd. Thank you for your quick response.
I checked and I realized the problem maybe is iCloud's mechanism.
I have 3 devices (real devices) with the same iCloud account. But only 1 device has this sync problem. I think the reason is "that device's iCloud settings" but I checked so many times and I didn't see any different setting.
It's so strange. I don't know. I will come back when I have more information.
Thank you so much.
